src/libs/ajax/GoogleStorage.ts

```diff
@@ -61,6 +63,22 @@ const withRequesterPays =
     return tryRequest();
   };
 
+const checkRequesterPaysError = async (response): Promise<RequesterPaysErrorInfo> => {
```

Comment:
right now, this module feels like the right home for these request-error related bits.